governance/votes/Executive Vote - April 4, 2024.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Template - [Executive Vote] Increase GSM Pause Delay, Increase Spark MetaMorpho Vault Maximum Debt Ceiling, Add Native Vaults to Debt Ceiling Breaker, SparkLend Proxy Spell, and TACO Resolutions - April 4, 2024
```

Comment:
We usually leave the spark proxy spell last, out of curiosity, why not this time?

Reply:
In general we agree it should be last but this ordering matches the sheet and it was too late to change it at the point we were reviewing. In future Proxy Spell actions should go at the end of the copy - can you add this to the style guide?